### PR TITLE
blog: mention Hemanth to styleText api

### PR DESCRIPTION
**Pull request by RafaelGSS**

### CODE REVIEW

**Comment by RafaelGSS** [comment]
@nodejs/nodejs-website should I open this PR against `legacy` too?

File: pages/en/blog/release/v21.7.0.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ const errorMessage = styleText('red', 'Error! Error!');
 console.log(errorMessage);
 ```
 
-Contributed by Rafael Gonzaga in [#51850](https://github.com/nodejs/node/pull/51850).
+Contributed by Rafael Gonzaga and Hemanth in [#51850](https://github.com/nodejs/node/pull/51850).
```
Comment (hemanth) [inline pages/en/blog/release/v21.7.0.md]: Can we please amend it to "Hemanth HM" ^ thank you.

**Comment by bmuenzenmeyer** [comment]
> @nodejs/nodejs-website should I open this PR against `legacy` too?

it will all be eventually correct - so doing it here is certainly future-friendly. if you want accuracy on the live post right now until the time the new site is live (measured in a short amount of time), you can open one there too. blog post content is up to you

**Review by bmuenzenmeyer — APPROVED** [review]
appeciate the desire for accuracy here

**Comment by bmuenzenmeyer** [comment]
fast-tracking, this is a small update pr

**Pull request merged**